package/src/components/SelectableList/v1/SelectableList.js

```diff
@@ -0,0 +1,140 @@
+      border-right: ${applyTheme("selectableListBorderStyle")} ${applyTheme("selectableListBorderColor")};
+      padding: ${applyTheme("selectableListItemPadding")};
+      .leftAligned {
+        justify-content: flex-start;
+        label {
+          font-weight: ${applyTheme("selectableListLabelFontWeight")}
+        }
+        .detail {
+          font-size: ${applyTheme("selectableListDetailFontSize")};
+          font-family: ${applyTheme("selectableListFontFamily")};
+          margin-left: 2px;
+        }
+      }
+      .amex,
+      .visa {
+        label {
+          position: relative;
+        }
+        span {
+          margin: ${applyTheme("selectableListIconMargin")}
+        }
+        label span:after {
+          left: ${applyTheme("selectableListIconLeft")};
+          position: absolute;
+          content: " ";
+          width: ${applyTheme("selectableListIconWidth")};
+          height: ${applyTheme("selectableListIconHeight")};
+          display: inline-block;
+          border-radius: ${applyTheme("selectableListBorderRadius")};;
+          border: ${applyTheme("selectableListBorderStyle")} ${applyTheme("selectableListBorderColor")};
+        }
+      }
+    }
+  }
+  .listAction {
+    border-bottom: ${applyTheme("selectableListBorderStyle")} ${applyTheme("selectableListBorderColor")};
+    border-left: ${applyTheme("selectableListBorderStyle")} ${applyTheme("selectableListBorderColor")};
+    border-right: ${applyTheme("selectableListBorderStyle")} ${applyTheme("selectableListBorderColor")};
+    padding: ${applyTheme("selectableListItemPadding")};
+    height: ${applyTheme("selectableListHeight")};
+    display: flex;
+    align-items: center;
+    box-sizing: border-box;
+    @media (max-width: 768px) {
+      height: ${applyTheme("selectableListHeightMobile")};
+    }
+  }
+  > *:last-child {
+    border-bottom-right-radius: ${applyTheme("selectableListBorderRadius")};
+    border-bottom-left-radius: ${applyTheme("selectableListBorderRadius")};
+    .wrapper:last-child {
+      border-bottom-right-radius: ${applyTheme("selectableListBorderRadius")};
+      border-bottom-left-radius: ${applyTheme("selectableListBorderRadius")};
+    }
+  }
+`;
+
+class SelectableList extends Component {
+  static propTypes = {
+    /**
+     * If you've set up a components context using @reactioncommerce/components-context
+     * (recommended), then this prop will come from there automatically. If you have not
+     * set up a components context or you want to override one of the components in a
+     * single spot, you can pass in the components prop directly.
+     */
+    components: PropTypes.shape({
+      /**
+       * Pass either the Reaction `SelectableItem` component or your own component
+       * that takes `items` props and uses them to render a single item.
+       */
+      SelectableItem: CustomPropTypes.component.isRequired
+    }).isRequired,
+    items: PropTypes.arrayOf(PropTypes.shape({
```

Comment:
@machikoyasuda add missing props:  `label` and `detail`, `icon`, `className` and their types. And determine which ones can be optional vs required.

Reply:
Actually @willopez it's best to include only props that this component actually uses. If it passes the rest on, we rely on the child component to validate the shape props that it uses.